tests/js/client/communication/test-communication.js

```diff
@@ -767,6 +746,7 @@ function GenericAqlSetupPathSuite(type) {
     },
 
     tearDownAll: function () {
+      reconnectRetry(primaryEndpoint, "_system", "root", "");
```

Comment:
if the helper methods do this reconnectRetry internally this Is not required at all